### PR TITLE
Update pachyderm

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
@jonandernovella @dwhitena @jdoliner

**Comment by jonandernovella** [comment]
@LaurentGoderre @jdoliner  https://github.com/helm/charts/pull/11244. I closed this PR because lots of ENV vars related to the storage backend seemed to change starting from pachyderm 1.8+. Take a look at the changes. It might be nice fixing this.

**Comment by LaurentGoderre** [comment]
Wouldn't it be nice to at least bump it to the latest of 1.7 while 1.8 is worked on? I lost a few days hitting bugs that had already been solved in later versions

**Comment by jonandernovella** [comment]
@LaurentGoderre There is no need for bumping the chart to use 1.7.10. You can simply change the pachd version tag.

**Comment by jonandernovella** [comment]
@LaurentGoderre Can you please make the following changes? 

1) Bump the chart version 
2) Replace the `RoleBinding` resource with `ClusterRoleBinding`
3) Remove all references to versions which are not `1.7.11`

**Comment by LaurentGoderre** [comment]
@jonandernovella done #1 and #3. What is the purpose of #2?

**Comment by LaurentGoderre** [comment]
Ok, done number #3

File: stable/pachyderm/README.md [hunk 1]
```diff
@@ -131,13 +131,24 @@ Alternatively, a YAML file that specifies the values for the parameters can be p
...
 ```
 
+Specifying a pachyderm version
+------------------------
+
+To specify a pachyderm version run the following command:
+
+```console
+$ helm install --namespace pachyderm --name my-release \
+-set pachd.image.tag=1.8.3,pachd.worker.tag=1.8.3 \
```
Comment (jonandernovella) [inline stable/pachyderm/README.md]: Here you need to also update the version.

File: stable/pachyderm/templates/pachd_rolebinding.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 {{- if .Values.rbac.create }}
 apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: RoleBinding
+kind: ClusterRoleBinding
```
Comment (jonandernovella) [inline stable/pachyderm/templates/pachd_rolebinding.yaml:4]: This change should make the pipeline parameter `coefficient` work again.

**Review by jonandernovella — CHANGES_REQUESTED** [review]
After you make these small changes, it should be mergeable!

File: stable/pachyderm/templates/pachd_rolebinding.yaml (hunk 2 above)
Comment (jonandernovella) [reply stable/pachyderm/templates/pachd_rolebinding.yaml:4]: Can you please test this?

Comment (LaurentGoderre) [reply stable/pachyderm/templates/pachd_rolebinding.yaml:4]: Is there an issue for this I can look at? This is not an issue I ran into.

File: stable/pachyderm/README.md (hunk 1 above)
Comment (jonandernovella) [reply stable/pachyderm/README.md]: @LaurentGoderre This still needs to be fixed

**Comment by LaurentGoderre** [comment]
Done!

**Comment by jonandernovella** [comment]
@jdoliner LGTM. Can you take a look?

**Pull request merged**